### PR TITLE
Adding ability to respond to commands and issue events

### PR DESCRIPTION
**Pull request by acron0**

### CODE REVIEW

**Comment by otfrom** [comment]
squash the commits to get rid of the WIP or give it a better msg.

File: src/migrators/cassandra/201606031548_init.clj [hunk 1]
```diff
@@ -10,11 +10,9 @@
      conn
      (hayt/create-table
       "events"
-      (hayt/column-definitions {:key              :text
-                                :id               :uuid
-                                :event_id         :uuid
-                                :event_version    :text
-                                :creator           :uuid
+      (hayt/column-definitions {:id               :uuid
```
Comment (otfrom) [inline src/migrators/cassandra/201606031548_init.clj:13]: hooray for having migrations stored with the PR!

Comment (acron0) [inline src/migrators/cassandra/201606031548_init.clj:13]: :)

**Comment by otfrom** [comment]
:shipit:

**Pull request merged**